### ISSUE TITLE
Can't Uninstall completely even it's BCUninstaller

### ISSUE THREAD

**Issue by wharjack** [corbindavenport/nexus-tools#73]
![image](https://github.com/corbindavenport/nexus-tools/assets/52787179/311d4c82-dd0a-4054-8c54-00d6c7ba4aa4)

**Comment by nitro9a** [comment]
Having the same issue.

**Comment by corbindavenport** [comment]
I'll try to find time to fix this soon, in the meantime deleting the `NexusTools` folder from the `%AppData%` folder in the File Explorer works.

You can also run this command in a Command Prompt to clean up the Nexus Tools item in Add/Remove Programs:

```
reg delete "HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Uninstall\NexusTools" /f
```

**Comment by nitro9a** [comment]
Thanks.  I found a registry key under uninstall for nexustools called noModify.  I changed it to 0, but I still get the same errors.   Was this intentional to prevent uninstalling or is that for something else?  Also wondering why changing it to 0 does nothing.

**Comment by corbindavenport** [comment]
I have a fix for this in the `dev` channel now, the main issue is that Windows doesn't allow executables to delete themselves or their parent directory while they are running (unlike macOS and Linux). I'm not sure why I didn't catch that before, but the uninstaller now creates and runs a small batch script to delete the NexusTools folder after everything else is done.

The `noModify` key just changes the main button in the Add and Remove Programs Control panel from "Uninstall/Change" to "Uninstall". The Control Panel is deprecated and the Windows Settings app says "Uninstall" either way, so I removed that key. I also set the icon to the `cmd.exe` icon so it looks a bit less generic.

![image](https://github.com/corbindavenport/nexus-tools/assets/3879063/301ebe6c-b96d-4472-be20-ab16a0baaa42)

![image](https://github.com/corbindavenport/nexus-tools/assets/3879063/a8830b80-2ecd-4396-a09d-1c29585ad794)

I need to fix a few other bugs and then I'll make a new release with these changes.
